source/mainnet/smart-contracts/tutorials/wCCD/wCCD-interacting.rst

```diff
@@ -155,12 +155,6 @@ You are ready now to invoke the ``balanceOf`` function with the following comman
 .. image:: ./images/wCCD_tutorial_4.png
     :width: 100 %
 
-.. note::
```

Comment:
Why remove this information?

Reply:
The exact same note is repeated at the start of this part of the tutorial, line 71. It felt weirdly repetitive when I read it.